src/projetos/SugestaoPage.tsx

```
@@ -21,37 +21,35 @@ import {
 import { useNavigate } from 'react-router-dom'
 import { AxiosError } from 'axios'
-import { MultiSelectProfessores } from '@/components/MultiSelectProfessores'
 import SugestaoService from './SugestaoService'
 import CursoService from '@/aluno/CursoService'
-import ProfessoresService from '@/utils/ProfessoresService'
 import UsuarioService from '@/servidor/UsuarioService'
 
 const formSchema = z.object({
     id: z.number().optional(),
-    titulo: z.string().min(2, {
+    titulo: z.string().min(10, {
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)